src/sentry/sentry_metrics/indexer/ratelimiters.py

```diff
@@ -19,15 +19,11 @@
 
 
 def _build_quota_key(use_case_id: UseCaseKey, org_id: Optional[OrgId] = None) -> str:
-    use_case_str = {
-        UseCaseKey.PERFORMANCE: "performance",
-        UseCaseKey.RELEASE_HEALTH: "releasehealth",
-    }[use_case_id]
 
     if org_id is not None:
-        return f"metrics-indexer-{use_case_str}-org-{org_id}"
+        return f"metrics-indexer-{use_case_id}-org-{org_id}"
     else:
-        return f"metrics-indexer-{use_case_str}-global"
+        return f"metrics-indexer-{use_case_id}-global"
 
```

Comment:
okay yeah, sorry the quota settings in the options are the same, but the namespace is what we need for the key. updated to use the namespace instead of the `use_case_id` and added a test

Reply:
I don't get it. IMO quotas should not be namespace specific but only based on the use_case_id. The same quota should be applied for all backends.

Reply:
the quotas are based on the `use_case_id`, the namespace is just used for the redis key.

Reply:
Ah ok, sorry you are right. I assumed quotas that code that records usage is separate to quotas codebase but looks like that might not be true.